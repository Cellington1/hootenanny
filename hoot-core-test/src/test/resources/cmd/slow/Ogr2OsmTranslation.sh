#!/bin/bash
set -e

export OUT=test-output/cmd/Ogr2OsmTranslation
rm -rf $OUT
mkdir -p $OUT
hoot --osm2shp "name,poi" test-files/OnePoint.osm $OUT/OnePoint.shp
hoot --ogr2osm translations/Identity.js $OUT/OnePointTranslated.osm $OUT/OnePointPoints.shp
hoot --is-match test-output/cmd/Ogr2OsmTranslation/OnePointTranslated.osm test-files/OnePoint.osm

# Make sure we can read all layers from a shapefile
rm -rf test-output/AllDataTypes.shp test-output/AllDataTypesA.zip
hoot --convert test-files/conflate/unified/AllDataTypesA.osm test-output/AllDataTypes.shp
cd test-output/AllDataTypes.shp/
zip -q ../AllDataTypesA.zip *
cd $HOOT_HOME
hoot --ogr2osm translations/Identity.js test-output/AllDataTypesA.osm /vsizip/./test-output/AllDataTypesA.zip/
hoot --stats test-output/AllDataTypesA.osm | grep -w 'Node Count\|Way Count\|Relation Count'
