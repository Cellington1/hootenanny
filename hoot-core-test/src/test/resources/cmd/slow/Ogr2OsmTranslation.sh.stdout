10:11:41.659 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 176) Reading: test-output/cmd/Ogr2OsmTranslation/OnePointPoints.shp OnePointPoints
Loading test-output/cmd/Ogr2OsmTranslation/OnePointPoints.shp OnePointPoints 0 / 1   
Reprojecting 0 / 1         
10:11:41.664 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::MergeNearbyNodes
10:11:41.664 INFO  .../core/ops/MergeNearbyNodes.cpp(  73) MergeNearbyNodes start
MergeNearbyNodes 1 elapsed: 0ms        
10:11:41.664 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::BuildingPartMergeOp
Ways: 0 / 0        
Relations: 0 / 0        
Combining Parts: 0 / 0        
Reprojecting 0 / 1         
10:11:41.994 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 176) Reading: /vsizip/./test-output/AllDataTypesA.zip/ Lines
Loading /vsizip/./test-output/AllDataTypesA.zip/ Lines 0 / 4   
10:11:41.995 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 176) Reading: /vsizip/./test-output/AllDataTypesA.zip/ Points
Loading /vsizip/./test-output/AllDataTypesA.zip/ Points 0 / 11   
10:11:41.997 INFO  ...p/hoot/core/cmd/Ogr2OsmCmd.cpp( 176) Reading: /vsizip/./test-output/AllDataTypesA.zip/ Polygons
Loading /vsizip/./test-output/AllDataTypesA.zip/ Polygons 0 / 10   
Reprojecting 0 / 122         
10:11:42.024 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::MergeNearbyNodes
10:11:42.024 INFO  .../core/ops/MergeNearbyNodes.cpp(  73) MergeNearbyNodes start
MergeNearbyNodes 0 1    MergeNearbyNodes 116 elapsed: 0ms        
10:11:42.024 INFO  .../cpp/hoot/core/ops/NamedOp.cpp(  56) Applying operation: hoot::BuildingPartMergeOp
Ways: 0 / 14        Ways: 14 / 14        
Relations: 0 / 0        
Combining Parts: 0 / 10        Combining Parts: 10 / 10        
Reprojecting 0 / 116         
Node Count	116
Way Count	14
Relation Count	0
